Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -527,7 +525,7 @@ def CutMD(*args, **kwargs):  # noqa: C901
         return out_names[0]
 
 
-_replace_signature(CutMD, ("\bInputWorkspace", "**kwargs"))
+CutMD.__signature__ = LazyFunctionSignature(alg_name="CutMD", include_self=False)
```

Comment:
Do you know why some algorithms need special treatment? E.g. `CutMD` and `RenameWorkspace`